tests/test_installer_backend.py

```diff
@@ -1,15 +1,41 @@
 import json
 import os
 import subprocess
+import uuid
 
 import passlib.hash
+import pytest
 
+import gen
+import gen.installer.aws
+import release
 from dcos_installer import backend
 from dcos_installer.config import Config, make_default_config_if_needed, to_config
 
 os.environ["BOOTSTRAP_ID"] = "12345"
 
 
+@pytest.fixture(scope='module')
+def config():
```

Comment:
Just as a note, discussed this offline previously, sadly this is the easiest way to get the fixtures here for now. Ideally we should have one copy.

@mellenburg any idea? (tried putting them in a common import but that blows up in fun ways like the upstream mixed with downstream tests  / fixture stuff you ran into)